### PR TITLE
[16.0][IMP] base_edifact: Support more EDIFACT format

### PR DESCRIPTION
**Pull request by thienvh332**
- Support "latin-1" format
- Get product fallback on SA if no EAN is given
- Fallback if address information is missing
- Add get price with Price qualifier is AAB
- support getting product description from the IMD segment

### CODE REVIEW

File: base_edifact/models/edifact.py [hunk 1]
```diff
@@ -70,7 +70,10 @@ def pydifact_obj(self, docu):
 
     @api.model
     def _loads_edifact(self, order_file):
-        interchange = Interchange.from_str(order_file.decode())
+        try:
+            interchange = Interchange.from_str(order_file.decode())
+        except UnicodeDecodeError:
+            interchange = Interchange.from_str(order_file.decode('latin-1'))
```
Comment (simahawk) [inline base_edifact/models/edifact.py]: I would add a TODO for the long term to use chardet. 
Eg: https://github.com/OCA/connector-interfaces/blob/16.0/connector_importer/utils/import_utils.py#L20

**Review by simahawk — COMMENTED** [review]
still draft?

File: base_edifact/models/edifact.py (hunk 1 above)
Comment (thienvh332) [reply base_edifact/models/edifact.py]: Thanks @simahawk , updated

**Review by simahawk — APPROVED** [review]
LG

**Pull request merged**